lib/block_view_types.go

```diff
@@ -2104,6 +2287,8 @@ type MessagingGroupMember struct {
 
 	// EncryptedKey is the encrypted messaging public key, addressed to the recipient.
 	EncryptedKey []byte
+
+	IsMuted bool
```

Comment:
With the new design, we should probably remove this from the `MessagingGroupEntry`